File: yuri/week3/easy.py
def solution(rsp):
    rsp = list(rsp)
    
    for i in range(len(rsp)):
        if rsp[i] == '2':
            rsp[i] = '0'
        elif rsp[i] == '0':
            rsp[i] = '5'
        else:
            rsp[i] = '2'
    
    return ''.join(rsp)

File: juha/week3/easy.js
function solution(rsp) {
  const win = {
    2: "0",
    0: "5",
    5: "2",
  };
  const Rsp = rsp.split("");
  const WinArray = Rsp.map((char) => {
    return win[char];
  });
  const answer = WinArray.join("");
  return answer;
}
